sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/DateExpressionsSuite.scala

```diff
@@ -46,7 +46,8 @@ class DateExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
   val jstId = Option(TimeZoneJST.getID)
 
   def toMillis(timestamp: String): Long = {
-    val tf = TimestampFormatter("yyyy-MM-dd HH:mm:ss", ZoneOffset.UTC)
+    val tf = TimestampFormatter(
+      "yyyy-MM-dd HH:mm:ss", ZoneOffset.UTC, needVarLengthSecondFraction = false)
```

Comment:
can we set the default value of `needVarLengthSecondFraction` as false?

Reply:
to avoid changes like this

Reply:
I changed the most used `apply` with default `needVarLengthSecondFraction` as false